canonical-web-and-design/canonicalwebteam.blog
# Change image URL

Find and replace any references to https://admin.insights.ubuntu.com/wp-content with https://ubuntu.com/wp-content.

Make the link replacement optional.

QA
--
1. Go to your local ubuntu.com repo. Enable the new blog changes for testing:
```bash
git remote add albert-fork git@github.com:Albert-Arthur/ubuntu.com.git  # get my fork
git fetch albert-fork  # fetch the branches
git checkout enable-transform-link  # checkout the branch

dotrun  # run website
```

Browse to `/blog`

The thumbnails and images should work as usual, and if you inspect element you will see that
the links point to `ubuntu.com/wp-content` instead of `admin.insights.ubuntu.com`

Similarly if you go to a blog page fx:
`/blog/dell-xps-13-developer-edition-with-ubuntu-20-04-lts-pre-installed-is-now-available`

The image will again point to `ubuntu.com/wp-content`.

## Fixes #2130 step 2
